frontend/hub/administration/repositories/components/RepositoryLabels.tsx

```diff
@@ -9,16 +9,20 @@ export function RepositoryLabels(props: { repository: Repository }) {
   if (Object.keys(repository.pulp_labels).length === 0) {
     return <TextCell text={t('None')} />;
   } else {
-    return Object.keys(repository.pulp_labels).map((label) =>
-      repository.pulp_labels[label] ? (
-        <Label readOnly key={label}>
-          {label}: {repository.pulp_labels[label]}
-        </Label>
-      ) : (
-        <Label readOnly key={label}>
-          {label}
-        </Label>
-      )
+    return (
+      <>
+        {Object.keys(repository.pulp_labels).map((label) =>
+          repository.pulp_labels[label] ? (
+            <Label readOnly key={label}>
+              {label}: {repository.pulp_labels[label]}
+            </Label>
+          ) : (
+            <Label readOnly key={label}>
+              {label}
+            </Label>
+          )
+        )}
+      </>
```

Comment:
this just wraps the list of components in a fragment

it.. doesn't change anything either? Unless it's a typing issue and something can't cope with `ReactElement[]` even tough `ReactNode` allows it?

Reply:
Change needed to fix typescript issues.